File: src/ecosistema/Locacion.java
package ecosistema;

public class Locacion {

    private String _nombre;

    public String getNombre() {
        return this._nombre;
    }

    public void setNombre(String aNombre) {
        this._nombre = aNombre;
    }

    @Override
    public String toString() {
        return "Locacion{" + "_nombre=" + _nombre + '}';
    }

    

}

File: src/ruby/Ecosistema/Locacion.rb
module Ecosistema
	public
	class Locacion

		# @ReturnType String
		public
		def getNombre()
			return @_nombre
		end

		# @ParamType aNombre String
		# @ReturnType void
		public
		def setNombre(aNombre)
			@_nombre = aNombre
		end

		public
		def initialize()
			# @AttributeType String
			@_nombre

			# Not yet implemented
		end
	end
end
